Dockerfile.windows

```diff
@@ -101,6 +104,24 @@ RUN cmake --build . --config Release; `
 
 # TODO: Do something with /work/out/bin/fluent-bit.{exe,dll}
 
+
+###############################################################################
+# Build JMX Project
+###############################################################################
+
+FROM base as javacontrib
+
+ADD https://javadl.oracle.com/webapps/download/AutoDL?BundleId=242044_3d5a2bb8f8d4428bbe94aed7ec7ae784 /local/jdk-8u251-windows-x64.exe
```

Comment:
Is this the right JDK?

Reply:
Manually validated on windows with correct JDK.